generate-err-logs.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
```

Comment:
should be `#!/bin/bash -e` so the script will stop if there is an error.

Reply:
Right, added